rskj-core/src/main/java/org/ethereum/datasource/KeyValueDataSourceUtils.java

```diff
@@ -67,7 +67,7 @@ public static DbKind getDbKindValueFromDbKindFile(String databaseDir) {
                 return DbKind.ofName(props.getProperty(KEYVALUE_DATASOURCE_PROP_NAME));
             }
 
-            return DbKind.LEVEL_DB;
+            return DbKind.ROCKS_DB;
```

Comment:
I guess we should also handle the case when db exists but dbKind file is missing (eg LevelDB created by default by some of the previous RSKj versions)

Reply:
You're right, I added code to check if the file exists and generate it if it does not.